packages/ui-components/src/components/range-dates-select-dropdown/range-dates-select-dropdown.tsx

```diff
@@ -90,11 +98,19 @@ export class KvRangeDatesSelectDropdown implements IRangeDatesSelectDropdown, IR
 		this.isOpen = openState;
 	};
 
+	private onSelectRangeDates = ({ detail: { payload: newSelectRangeDates } }: CustomEvent<ISelectRangeDates>) => {
```

Comment:
This seems to be closing the dropdown right after selecting both dates, is this behavior expected? If so, can we make it togglable?